streamwebs_frontend/streamwebs/tests/models/test_water_quality.py

```diff
@@ -9,17 +9,16 @@
 
 
 class WaterQualityTestCase(TestCase):
-
     def setUp(self):
         self.expected_fields = {
             'site': models.ForeignKey,
             'site_id': models.ForeignKey,
-            'DEQ_dq_level': models.CharField,  # DEQ Data Quality Level
+            'DEQ_dq_level': models.CharField,
             'date': models.DateField,
             'school': models.CharField,
             'latitude': models.DecimalField,
             'longitude': models.DecimalField,
-            'fish_present': models.BooleanField,
+            'fish_present': models.CharField,
```

Comment:
Yeah, I think the BooleanField used different syntax for the default value, and I thought it would be less confusing to just be consistent with the other radio inputs.